admin/javascript/src/reducer-register.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * The register of Redux reducers.
+ * @private
+ */
+const register = {};
+
+/**
+ * The central register of Redux reducers for the CMS.
```

Comment:
Since there can only be one "root reducer" in Redux that's composed of other reducers, it makes sense to only have one central `ReducerRegister` for any admin code that wants to influence state changes, regardless of module. So the documentation is correct stating "for the CMS".